### DISCUSSION TITLE
Sample wallet doesnt support `cosmos_getAccounts` and listing rejected

### DISCUSSION THREAD

**Discussion by williamchong** [WalletConnect/walletconnect-monorepo#2860] [Developer Support] [answered]
Project ID: e110ac49451fee41d5bcda1b0dfdb94e
<img width="1233" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/6198816/4024e704-af56-4a4e-8956-d95a0476d191">

My dApp is running well live on https://liker.land , supporting cosmos ecosystem wallets running walletconnect v2
However the [sample wallet](https://react-web3wallet.vercel.app/walletconnect) used for explorer listing does not support the cosmos spec method `cosmos_getAccounts`, nor is deriving the proper bech32 address using `like` prefix properly
<img width="490" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/6198816/8fd737d1-eadd-4be9-a9a3-6a37aa10f0a9">

Is this why the listing was rejected? If thats the reason how can i proceed?

**Comment by JeroldG** [comment, marked as answer]
Hi there, I have now approved your listing and able to see V2 metric data
